packages/api/src/chats/usecases/upload-documents-to-chat.usecase.ts

```diff
@@ -59,8 +59,6 @@ export class UploadDocumentsToChatUsecase implements Usecase {
         )
       );
     }
-
-    this.aiService.invalidateAgentCache(roomId);
```

Comment:
This needs to be tested out with the following test:
- create room A
- upload doc A
- ask question about doc A
- upload doc B
- ask question about doc B

If everything is correct, then all is good.

Reply:
Already tested it, seems to work as expected.